pkg/process/checks/process_events_linux_test.go

```diff
@@ -0,0 +1,231 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build linux
+// +build linux
+
+package checks
+
+import (
+	"context"
+	"io"
+	"testing"
+	"time"
+
+	payload "github.com/DataDog/agent-payload/v5/process"
+	"github.com/DataDog/datadog-go/v5/statsd"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/DataDog/datadog-agent/pkg/process/config"
+	"github.com/DataDog/datadog-agent/pkg/process/events"
+	"github.com/DataDog/datadog-agent/pkg/process/events/model"
+	"github.com/DataDog/datadog-agent/pkg/security/api"
+	"github.com/DataDog/datadog-agent/pkg/security/api/mocks"
+	secmodel "github.com/DataDog/datadog-agent/pkg/security/secl/model"
+)
+
+type eventTestData struct {
+	rawEvent     *model.ProcessMonitoringEvent
+	payloadEvent *payload.ProcessEvent
+}
+
+func parseRFC3339Time(t *testing.T, s string) time.Time {
```

Comment:
Nit: `t.Helper()` here?

Reply:
Thanks for catching it! Updated on https://github.com/DataDog/datadog-agent/pull/12390/commits/2b962fa19b7d8f6cde4a18e387c7693a52d97592.